aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -95,6 +95,10 @@
 
 CONFIG_DEFAULT = 'production/application-sfbay/base.conf'
 
+DEFAULT_STUCK_GUARD_MIN_CPU_USAGE = "5"
+
+DEFAULT_STUCK_GUARD_MAX_INACTIVE_TIME_INTERVAL = "10"
```

Comment:
These default values are not the same as in the `aws/build.gradle`. Which values are more appropriate? CPU usage is measured in percent or fraction of 1.0?

Reply:
These are just more safe values in case the branch did not get an update.